hls4ml/templates/vivado/nnet_utils/nnet_activation_stream.h

```diff
@@ -341,6 +341,313 @@ void softmax(hls::stream<data_T> &data, hls::stream<res_T> &res){
     }    
```

Comment:
These changes seem to not be from your codes, can you maybe clone my branch (`onnx_dev_pytorch`) and make changes based on this branch? (instead of doing it from master)